go.mod

```diff
@@ -1,12 +1,12 @@
 module github.com/virtual-kubelet/virtual-kubelet
 
-go 1.20
+go 1.21
```

Comment:
Is the go version upgrade required for the fix? If not, it'd be better to separate into a different PR